src/main/java/se/uu/elephas/model/Product.java

```diff
@@ -29,6 +29,10 @@
 
     private float discount;
 
+    @OneToOne(mappedBy = "product")
```

Comment:
I don't understand how this works, why is it OneToOne? I guess many different orders can have the same product.

Reply:
Yes indeed but I thought that each order has its own order items and each order item is a specific product. So, the relationship between order item and product is OneToOne